src/sources/metadatasourcetaglib.cpp

```diff
@@ -630,6 +630,15 @@ class SafelyWritableFile final {
         // See also: https://bugs.launchpad.net/mixxx/+bug/1815305
         DEBUG_ASSERT(m_origFileName.isNull());
         DEBUG_ASSERT(m_tempFileName.isNull());
+        QFileInfo origFileInfo(origFileName);
+        if (!origFileInfo.isWritable()) {
```

Comment:
Lets either make the temporary const or simply use the inlined expression`QFileInfo(origFileName).isWritable()`. I would prefer the latter.